google/cloud/functions/framework.h

```diff
@@ -12,34 +12,30 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#ifndef FUNCTIONS_FRAMEWORK_CPP_GOOGLE_CLOUD_FUNCTIONS_INTERNAL_FRAMEWORK_H
-#define FUNCTIONS_FRAMEWORK_CPP_GOOGLE_CLOUD_FUNCTIONS_INTERNAL_FRAMEWORK_H
+#ifndef FUNCTIONS_FRAMEWORK_CPP_GOOGLE_CLOUD_FUNCTIONS_FRAMEWORK_H
+#define FUNCTIONS_FRAMEWORK_CPP_GOOGLE_CLOUD_FUNCTIONS_FRAMEWORK_H
 
-#include "google/cloud/functions/internal/user_functions.h"
+#include "google/cloud/functions/user_functions.h"
 #include "google/cloud/functions/version.h"
 #include <functional>
 
-namespace google::cloud::functions_internal {
+namespace google::cloud::functions {
 inline namespace FUNCTIONS_FRAMEWORK_CPP_NS {
 
 /**
  * Run the given function, invoking it to handle HTTP requests.
  *
  * Starts a HTTP server at the address and listening endpoint described by
- * @p argv, invoking @p handler to handle any HTTP request. Note that we do not
- * expect the application to use this function directly (it is in
- * `functions_internal` after all), instead, our build scripts should create
- * a `main()` function that calls this, as shown in the example:
+ * @p argv, invoking @p handler to handle any HTTP request.
```

Comment:
Do you think it's worth documenting that `Run()` is only intended to be called by users when they're trying to test their code outside of the normally built docker image? Otherwise, users may be mislead into thinking that they need to call this at some point in their real/production code.

Reply:
Done.